src/Pie.js

```diff
@@ -18,37 +18,71 @@ function createPath(cx, cy, r, startAngle, arcAngle) {
       r,
       startAngle,
       startAngle + arcAngle,
-    )
+    );
   } else {
-    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle))
-    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1)
+    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle));
+    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1);
   }
-  return p
+  return p;
 }
 
-const ArcShape = ({ radius, width, color, strokeCap,startAngle, arcAngle }) => {
+const ArcShape = ({ radius, width, color, strokeCap, startAngle, arcAngle }) => {
   const path = createPath(
     radius,
     radius,
     radius - width / 2,
     startAngle / 180 * Math.PI,
-    arcAngle / 180 * Math.PI
-  )
-  return <Shape d={path} stroke={color} strokeWidth={width} strokeCap={strokeCap} />
-}
+    arcAngle / 180 * Math.PI,
+  );
+  return <Shape d={path} stroke={color} strokeWidth={width} strokeCap={strokeCap} />;
+};
 
-const RingShape = props =>
```

Comment:
I tested removing `RingShape`, everything appears to be as expected. @nihgwu , what was the specific scenario that required this originally?

Reply:
https://github.com/nihgwu/react-native-pie/commit/86adf51339854ef3dc50df8cef6d12afb9df7b82

Reply:
The original code snippet from the facebook PR looks like the following with these changes (ignore formatting):
<img width="360" alt="Screen Shot 2020-01-08 at 3 44 46 PM" src="https://user-images.githubusercontent.com/10931905/72018751-d82d8380-322d-11ea-8b1a-43134e57c833.png">
So we are good